libr/core/cbin.c

```diff
@@ -2883,6 +2883,10 @@ static int bin_classes(RCore *r, int mode) {
 			r_list_foreach (c->methods, iter2, sym) {
 				char *mflags = r_core_bin_method_flags_str (sym->method_flags, mode);
 				char *cmd = r_str_newf ("\"f method%s.%s.%s = 0x%"PFMT64x"\"\n", mflags, c->name, sym->name, sym->vaddr);
+				r_str_replace_char (cmd, ' ', '_');
+				cmd[2] = ' ';
+				char *eq = strrchr (cmd, '=');
+				*(eq - 1) = *(eq + 1) = ' ';
```

Comment:
if string starts with = this is an oobwrite

Reply:
use e[] instead of *(-)

Reply:
> if string starts with = this is an oobwrite

ok 0898dfc6fbaac57bf88ec21bb133d940fc38133f

> use e[] instead of *(-)

can't do `eq[-1]` ... the alternative:

```c
eq--;
eq[0] = eq[2] = ' ';
```

is more confusing.

Reply:
oops can do `eq[-1]` apparently